dlpy/utils.py

```diff
@@ -1077,10 +1121,11 @@ def create_object_detection_table(conn, data_path, coord_type, output,
     unix_type = server_type.startswith("lin") or server_type.startswith("osx")
     # check if local and server are same type of OS
     # in different os
+    need_to_parse = True
     if (unix_type and local_os_type.startswith('Win')) or not (unix_type or local_os_type.startswith('Win')):
         if local_path is None:
-            raise ValueError('local_path must be specified when your server is on {} OS and local '
-                             'python is on {} OS'.format(server_type.split('.')[0].capitalize(), local_os_type))
+            warnings.warn('The txt files in data_path are used as annotation files.', RuntimeWarning)
```

Comment:
does this work in multiple python versions. can you please convert this to just print statement - that should be fine for now